docs/release-notes/2.18.0.md

```diff
@@ -0,0 +1,93 @@
+
+* [PMM-7509](https://jira.percona.com/browse/PMM-7509): Integrated Alerting: Pagination for Alerts list
+
+
+
+## Improvements
+
+* [PMM-8029](https://jira.percona.com/browse/PMM-8029): DBaaS: PSMDB 1.8v operator support
+* [PMM-7548](https://jira.percona.com/browse/PMM-7548): Integrated Alerting: Disable edit and delete buttons for templates with source USER_FILE
+
+
+
+## Bugs Fixed
+
+* [PMM-8132](https://jira.percona.com/browse/PMM-8132): DBaaS: Monitoring of PXC cluster stops working after upgrade from 2.16.0 to 2.18.0
+* [PMM-8127](https://jira.percona.com/browse/PMM-8127): DBaaS: DB Clusters disappear when upgrading from 2.16.0 to 2.18.0
+* [PMM-8058](https://jira.percona.com/browse/PMM-8058): IA: Firing alerts disappear after docker restart
+* [PMM-8069](https://jira.percona.com/browse/PMM-8069): IA: Alert template .yaml is not recognized
+* [PMM-8053](https://jira.percona.com/browse/PMM-8053): 'An unexpected error happened' when accessing Settings page as non-admin
+* [PMM-7916](https://jira.percona.com/browse/PMM-7916): DBaaS: Wrong required resources when editing a cluster
+* [PMM-7782](https://jira.percona.com/browse/PMM-7782): Creating new notifications, while older ones are vanishing.
+* [PMM-7941](https://jira.percona.com/browse/PMM-7941): mongodb_exporter provides a wrong replication status value
+* [PMM-5350](https://jira.percona.com/browse/PMM-5350): 500 Internal Server Error due to file permissions
+* [PMM-7302](https://jira.percona.com/browse/PMM-7302): The attribute max_alerts is not valid for webhook_configs for AlertManager version included in PMM v2.13.0
+* [PMM-7224](https://jira.percona.com/browse/PMM-7224): Instance Overview dashboards behave inconsistently
+* [PMM-6864](https://jira.percona.com/browse/PMM-6864): PMM2 Oplog Recovery Window dashboard is broken (Thanks to Clyde Shillingford for reporting this issue)
+* [PMM-6543](https://jira.percona.com/browse/PMM-6543): Users logged out of PMM after some period
+* [PMM-7910](https://jira.percona.com/browse/PMM-7910): MongoDB Query Collector never recovers from failed cursor (Thanks to Yann Rouillard for reporting this issue)
+* [PMM-6890](https://jira.percona.com/browse/PMM-6890): The "Environment" filter not working (Thanks to Daniel Guzman Burgos for reporting this issue)
+* [PMM-6451](https://jira.percona.com/browse/PMM-6451): Passing parameters between QAN and dashboards
+* [PMM-5368](https://jira.percona.com/browse/PMM-5368): "Failed to get PMM Server parameter" after config. (Thanks to Martin Wittwer for reporting this issue)
+* [PMM-5135](https://jira.percona.com/browse/PMM-5135): Query Example is often empty (Thanks to Mikhail Solovyev for reporting this issue)
+* [PMM-7673](https://jira.percona.com/browse/PMM-7673): IA: Actions column is transparent
+* [PMM-8123](https://jira.percona.com/browse/PMM-8123): Custom Queries Stop working after Upgrade PMM-Server to 2.18.0
+* [PMM-8083](https://jira.percona.com/browse/PMM-8083): PMM-Agent Config is failing
```

Comment:
Seems to be an internal issue only...why would the world care?

Reply:
🤷‍♂️  looks like an Internal task to me too. Can't understand it by description from user side

Reply:
@denisok ?